.prettierrc

```diff
@@ -2,7 +2,6 @@
   "tabWidth": 2,
   "singleQuote": false,
   "trailingComma": "es5",
-  "printWidth": 100,
```

Comment:
Modifications to the Prettier config should be in a specific PR, or you should apply Prettier to all source files. This would avoid in the future to have style-specific modifications just because of the Prettier config that have changed.